README.md

```diff
@@ -17,6 +17,20 @@ To install or update to latest version of CLI use following script:
 curl -o- https://raw.githubusercontent.com/redhat-developer/app-services-cli/main/scripts/install.sh | bash 
 ```
 
+## RHOAS Container Image
+
+To install or update [image](https://github.com/redhat-developer/app-services-tools) containing RHOAS CLI:
+
+```shell
+docker pull quay.io/rhoas/tools
+```
+
+Running the image:
+
+```shell
+docker run -ti --rm --name rhoas-devsandbox --entrypoint /bin/bash quay.io/rhoas/tools
```

Comment:
Do we need container name (if container exist command will fail)

Reply:
It doesn't seem to be failing.
Have used the command specified in README of [app-services-tools](https://github.com/redhat-developer/app-services-tools#running-image)

Reply:
+1 lets leave it like this